services/appveyor/appveyor.js

```diff
@@ -42,14 +42,14 @@ module.exports = class AppVeyor extends BaseService {
     };
   }
 
-  static getExamples() {
+  static get examples() {
     return [
       {
-        uri: '/appveyor/ci/gruntjs/grunt',
+        previewUri: '/appveyor/ci/gruntjs/grunt',
       },
       {
-        name: 'Branch',
-        uri: '/appveyor/ci/gruntjs/grunt/master',
+        title: 'branch',
+        previewUri: '/appveyor/ci/gruntjs/grunt/master',
```

Comment:
This is a minor duplication, but given we define a base URI pattern in the service class I wonder if there is a way we could define the examples here as `gruntjs/grunt/master` or `{ 'repo': 'gruntjs/grunt', 'branch': 'master' }` without needing to repeat `/appveyor/ci` in each example?

Reply:
Yup, that makes a lot of sense!

Reply:
I pushed an update. Could you take a look?

It's cool that you brought up `{ 'repo': 'gruntjs/grunt', 'branch': 'master' }`. I've started some experimentation with [path-to-regexp](https://github.com/pillarjs/path-to-regexp) which has a [compile function](https://github.com/pillarjs/path-to-regexp#compile-reverse-path-to-regexp) that will help with this. I didn't quite get it working but my progress was promising!